docker/Dockerfile-backend

```diff
@@ -0,0 +1,20 @@
+#
+# Build the DronecodeSDK and launches the backend bin
+#
+# Author: Ramón Roche <julian@oes.ch>
+#
+
+FROM dronecode/dronecode-sdk-base:latest
+MAINTAINER Ramón Roche <mrpollo@gmail.com>
+
+ENV BUILD_BACKEND 1
+WORKDIR "/home/user/DronecodeSDK"
+ADD [".", "/home/user/DronecodeSDK"]
+
+# ADD can sometimes copy build/
+# lets make sure we have a clean build
+RUN mkdir build \
+	&& make clean
+RUN make default
```

Comment:
I think you need to do:
`RUN make default BUILD_BACKEND=1 default`

Reply:
And to build that you probably also need some more dependencies, e.g. golang.
See: https://sdk.dronecode.org/en/contributing/build.html#build_backend

Reply:
There is `ENV BUILD_BACKEND 1` above, but I would also add it to this command instead of globally.